Extension/package.nls.json

```diff
@@ -1,4 +1,10 @@
 {
+    "c_cpp.subheaders.intelliSense.title" : "IntelliSense",
+    "c_cpp.subheaders.codeAnalysis.title" : "Code Analysis",
+    "c_cpp.subheaders.formatting.title" : "Formatting",
+    "c_cpp.subheaders.doxygen.title" : "Doxygen",
```

Comment:
Is "Doxygen" a brand name or product name that can be translated? Should we use a title that's more generic like "Code Documentation"

Reply:
It's a product name that can't be translated. Yeah, "Code Documentation" might be better for new users who don't know what Doxygen is. And theoretically it could have other code documentation settings in the future for XML comments, etc.